src/event-stream/event-server.ts

```diff
@@ -911,101 +910,96 @@ export async function startEventServer(opts: {
 
   app.post(
     '/new_block',
-    asyncHandler(async (req, res, next) => {
+    asyncHandler(async (req, res) => {
       try {
-        const blockMessage: CoreNodeBlockMessage = req.body;
+        const blockMessage: CoreNodeBlockMessage = JSON.parse(req.body);
         await messageHandler.handleBlockMessage(opts.chainId, blockMessage, db);
         if (blockMessage.block_height === 1) {
           await handleBnsImport(db);
         }
+        await handleRawEventRequest(req);
         res.status(200).json({ result: 'ok' });
-        next();
       } catch (error) {
         logger.error(error, 'error processing core-node /new_block');
         res.status(500).json({ error: error });
       }
-    }),
-    handleRawEventRequest
+    })
```

Comment:
This change where `handleRawEventRequest` is called fixes the sql inconsistency bug that likely caused event ingestion to succeed even if the payload failed to write into the raw events table.